README.md

```diff
@@ -55,15 +55,17 @@ You can install the package via composer:
 
 ### Register Service Provider
 
-Add the service provider in `config/app.php`:
+Add the service provider in `bootstrap/providers.php`:
 
 ```php
 Usamamuneerchaudhary\Commentify\Providers\CommentifyServiceProvider::class,
 ```
 
 ### Run Migrations
 
-Once the package is installed, you can run migrations,
+Copy the migrations from the migrations folder first to your migrations.
```

Comment:
Hello @LUCASFRANKINC , don't think we need to copy over migrations like this. Once service provider is registered. Migrations can be run normally from the pacakge